conans/client/rest/rest_client_v1.py

```diff
@@ -138,23 +138,27 @@ def _upload_package(self, pref, files_to_upload, retry, retry_wait):
         if self._matrix_params:
             urls = self.router.add_matrix_params(urls)
         logger.debug("Requesting upload urls...Done!")
-        self._upload_files(urls, files_to_upload, self._output, retry, retry_wait)
+        short_pref_name = "%s:%s" % (pref.ref, pref.id[0:4])
+        self._upload_files(urls, files_to_upload, self._output, retry, retry_wait,
+                           ref_or_package=str(short_pref_name))
 
-    def _upload_files(self, file_urls, files, output, retry, retry_wait):
+    def _upload_files(self, file_urls, files, output, retry, retry_wait, ref_or_package=None):
```

Comment:
Maybe argument can be called something like "display_name" or "print_name" or "description"